pepper-apis/docs/specification/src/endpoints/user.studies.activity.answers.yml

```diff
@@ -64,8 +64,8 @@ patch:
       description: missing credentials or temporary user is invalid
     404:
       $ref: '../pepper.yml#/components/responses/ActivityAnswersPatchNotFoundResponse'
-    409:
-      $ref: '../pepper.yml#/components/responses/ActivityAnswersPatchExistsResponse'
+    500:
+      $ref: '../pepper.yml#/components/responses/ActivityAnswersMoreThanOneAnswerResponse'
```

Comment:
I don't think we need this? It'll just be a general error response, as specified by `default` below?

Reply:
Done.

Reply:
Can we remove the `500` and the `ActivityAnswersMoreThanOneAnswerResponse`? It's the same as the `default` below.

Reply:
You're right, it makes sense to remove the `500` section. Done.